lab-anthony/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "lab-anthony",
+  "version": "0.0.1",
+  "description": "",
+  "main": "entry.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "./node_modules/eslint/bin/eslint.js .",
+    "watch": "./node_modules/webpack-dev-server/bin/webpack-dev-server.js --inline --hot"
+  },
+  "author": "ascasson",
+  "license": "MIT",
+  "devDependencies": {
+    "angular": "^1.5.7",
```

Comment:
Separate your dependencies and dev dependencies. Angular should not be listed here.